nixos/doc/manual/release-notes/rl-1709.xml

```diff
@@ -187,6 +187,17 @@ rmdir /var/lib/ipfs/.ipfs
       have therefore been removed.
     </para>
   </listitem>
+  <listitem>
```

Comment:
I am going to be super unhelpful: xD
The way this is phrased is somehow strange but I can't point out why... Maybe somebody else can chime in?